pkg/apps/blackduck/components/rc/jobrunner/v1/component.go

```diff
@@ -35,11 +35,10 @@ func (c *BdReplicationController) GetRc() (*components.ReplicationController, er
 
```

Comment:
Probably should verify containerConfig isn't nil.  Technically could have key with nil value, possibly in error/edge cases, so should verify containerConfig isn't nil